sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
-## 4.2.0-beta.4 (Unreleased)
+## 4.2.0 (2021-06-08)
+
+### New Features
+
+- Added support for the 7.2 version of the Key Vault service API.
```

Comment:
Is this surface/changelog listed for the API somewhere? Or is it just kind of understood by consumers how to find this information?

Reply:
(and with that said - what changed in the package that might be interesting for users? New APIs? Better stuff?, etc..?). Probably worth having some highlights in here.

Reply:
For certificates specifically there isn't much in new features, but for packages that did have meaningful changes I added highlights for new features. Here it's literally _just_ supporting 7.2. 

What I'm thinking now is that using 7.2 isn't really a _feature_ and doesn't give the users anything interesting for certificates so I will remove this section. Thanks!